delegation/interactor/validatorKey1.pem

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PRIVATE KEY for 62b55cc67494ea9488e4406a2ebd52ffb2f8d58ea08eb37c15a3b5c7398f9b4209f9fa8b93c9d09e6f5a2aef06adc20d412c4ebbaccbc88556f2d05ff519243c1411a4847efb45887d2b6c7ae0e304da006f7d3565d9aa505fe7b2e170cdaf09-----
+YjEzNDA2YjM1YTY4ODlkNWM4M2IwMDliZDFmNTA4N2FlMjBjZTM1ZDMwZDFkZGJj
```

Comment:
do we really need these ? there should be no private key in the repo. also, there is no need for this for testing purposes.

Reply:
I just imported these from delegation interactor as they were in framework. I'll check if we can make a cleaner import